### PR TITLE
[stable/fluentd] Automatically roll deployment when configmap change

### PR DESCRIPTION
**Pull request by f84anton**
#### What this PR does / why we need it:

Trigger deployment update when configmap change

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/fluentd/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Fluentd Elasticsearch Helm chart for Kubernetes.
 icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
 name: fluentd
-version: 1.6.0
+version: 1.6.1
```
Comment (Miouge1) [inline stable/fluentd/Chart.yaml]: This is probably more of a feature than a bugfix, so 1.7.0 ?

**Comment by f84anton** [comment]
@Miouge1, agree with you, it's a feature.

**Comment by Miouge1** [comment]
@f84anton thank you for the contribution. It looks good to me.

/lgtm
/approve

**Pull request merged**